packages/donation-form/src/donation-form.ts

```diff
@@ -55,16 +56,14 @@ export class DonationForm extends LitElement {
 
   @property({ type: Object }) contactForm?: ContactForm;
 
+  @property({ type: Array }) amountOptions: number[] = defaultDonationAmounts;
+
   @property({ type: Boolean }) private creditCardVisible = false;
 
   @property({ type: Boolean }) private contactFormVisible = false;
 
   @property({ type: Boolean }) private donationInfoValid = true;
 
-  @property({ type: Boolean }) private contactInfoValid = false;
-
-  @property({ type: Boolean }) private hostedFieldsValid = false;
-
```

Comment:
QUESTION: I assume this deletion is just some cleanup? (Not directly relevant to ticket?)

Reply:
Oh yeah these were unrelated.. just noticed they weren't being used and removed them.